include/effectmanager.h

```diff
@@ -96,6 +95,61 @@ class EffectManager : public IJSONSerializable
         }
     }
 
+    void LoadJSONAndMissingEffects(const JsonArrayConst& effectsArray)
+    {
+        std::set<int> loadedEffectNumbers;
+
+        // Create effects from JSON objects, using the respective factories in g_EffectFactories
+        auto& jsonFactories = g_EffectFactories.GetJSONFactories();
+
+        for (auto effectObject : effectsArray)

```

Comment:
auto & perhaps?

Reply:
Here I'm following the examples in ArduinoJson's documentation. As all JSON "objects" are in fact elaborate pointers, it doesn't hurt much anyway.